### PR TITLE
Update free-courses-pt_BR.md

### PR DESCRIPTION
**Pull request by lenalightning**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -137,6 +137,7 @@
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 4 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dkcVCk2Bn_fdVQ81Fkrh6WT) - Gustavo Guanabara (Curso em Vídeo)
 * [Introdução à linguagem CSS](https://www.udemy.com/introducao-a-linguagem-css/) - Diego Mariano (Udemy)
 * [Introdução à Linguagem HTML](https://www.udemy.com/introducao-a-linguagem-html/) - Diego Mariano (Udemy)
+* [Discover](https://app.rocketseat.com.br/discover) - Rocketseat
```
Comment (ShivamPathak99) [inline courses/free-courses-pt_BR.md:140]: Arrange the contents in alphabetical order.
[Discover]    **should come before**   [Introdução à linguagem CSS]

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -181,6 +182,7 @@
 * [Curso Intensivo de Next.js & React](https://www.cod3r.com.br/courses/curso-intensivo-next-react) - Leonardo Leitão (Cod3r)
 * [Curso Starter ES6, Node, React e React Native](https://app.rocketseat.com.br/discover/courses) - Rocketseat
 * [Fast & Furious](https://www.youtube.com/playlist?list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts
+* [Discover](https://app.rocketseat.com.br/discover) - Rocketseat
```
Comment (ShivamPathak99) [inline courses/free-courses-pt_BR.md:185]: Arrangement in alphabetical order is required.

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -247,6 +249,7 @@
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab (YouTube)
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
 * [RESTful com Node.js e Restify](https://www.youtube.com/playlist?list=PLy5T05I_eQYO5Y3S3kVqBxQzkUNllPazF) - Codecasts (YouTube)
+* [Ignite](https://www.rocketseat.com.br/ignite) - Rocketseat
```
Comment (ShivamPathak99) [inline courses/free-courses-pt_BR.md:252]: Arrangement in alphabetical order is required.

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -335,6 +339,7 @@
 * [Mini-curso de TypeScript](https://www.youtube.com/playlist?list=PLlAbYrWSYTiPanrzauGa7vMuve7_vnXG_) - Willian Justen Curso (YouTube)
 * [TypeScript - Aprendendo Junto](https://www.youtube.com/playlist?list=PL62G310vn6nGg5OzjxE8FbYDzCs_UqrUs) - DevDojo (YouTube)
 * [TypeScript, o início, de forma prática](https://www.youtube.com/watch?v=0mYq5LrQN1s) - Rocketseat, Diego Fernandes (YouTube)
+* [Ignite](https://www.rocketseat.com.br/ignite) - Rocketseat
```
Comment (ShivamPathak99) [inline courses/free-courses-pt_BR.md:342]: Arrangement in alphabetical order is required.

File: courses/free-courses-pt_BR.md (hunk 2 above)
Comment (eshellman) [reply courses/free-courses-pt_BR.md:185]: not a direct link - can't use

**Comment by eshellman** [comment]
these aren't links to the course - can't use them

**Pull request closed without merging**